src/bda/plone/orders/browser/bookings.py

```diff
@@ -166,20 +167,20 @@ def render_buyable_uid(self, colname, record):
         bookings_quantity = self.render_bookings_quantity(colname, record)
         bookings_total_sum = self.render_bookings_total_sum(colname, record)
         value = \
-            '<tr class="group_buyable">' \
-            '<td colspan="13">' + '<p>' + title + '</p>' +\
-            '<span>' +\
+            u'<tr class="group_buyable">' \
+            u'<td colspan="13">' + u'<p>' + safe_unicode(title) + u'</p>' +\
+            u'<span>' +\
             translate(
                 _("bookings_quantity", default=u"Bookings quantity"),
                 self.request
             ) \
-            + ': ' + str(bookings_quantity) + '</span>' \
-            '<span>' +\
+            + u': ' + safe_unicode(bookings_quantity) + u'</span>' \
+            u'<span>' +\
             translate(
                 _("bookings_total_sum", default=u"Bookings total sum"),
                 self.request
             ) \
-            + ': ' + str(bookings_total_sum) + '</td></tr>'
+            + u': ' + bookings_total_sum + u'</td></tr>'
```

Comment:
maybe refactor that to make use multi line strings and the format string method?
```u"""blabla""".format(...)```